app/src/test/java/com/taskruler/ActivityTests.kt

```diff
@@ -109,11 +109,9 @@ class ActivityTests {
     }
 
     private fun deleteTask() {
-        var removeTask = addActivities
+        val removeTask = addActivities
 
-        if (removeTask != null) {
-            removeTask.drop(0)
-        }
+        removeTask?.drop(0)
```

Comment:
Use the elvis operator when testing if a var is null for more concise code.
[Elvis Operator](https://kotlinlang.org/docs/null-safety.html#elvis-operator)

Reply:
Good.